src/main/java/apoc/warmup/Warmup.java

```diff
@@ -39,13 +39,23 @@ public Warmup() {
         int relsPerPage = pageSize / RelationshipRecordFormat.RECORD_SIZE;
         long nodesTotal = Util.nodeCount(db);
         long relsTotal = Util.relCount(db);
+        long maxNodeId = Util.runNumericQuery(db, "MATCH (n) RETURN max(id(n)) AS result", null);
```

Comment:
I'd rather use this again, like the original implementation.

https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/25/files#diff-2a0b8c8ad695da883dab5f279def2112R50

This is instant. No query required.
Perhaps we should just revert to @SaschaPeukert's original implementation which is much more efficient on huge stores.